### PR TITLE
#165675656 Hosting Application on Heroku

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

-  Hosting Application on Heroku

#### Description of Task to be completed?
-  The app should be hosted on Heroku such that it can be accessed from anywhere in the world

#### How should this be manually tested?
  The hosted application on develop is [link](https://authors-haven-dev.herokuapp.com/)
  The hosted application on master link is  [link](https://ah-frontend-master.herokuapp.com/) 

#### What are the relevant pivotal tracker stories?
[ #165675656](https://www.pivotaltracker.com/story/show/165675656)

#### Screen shots

- Web Application
Screenshot 2019-04-29 at 15 15 59

- Pipelines
Screenshot 2019-05-02 at 15 42 46

### CODE REVIEW

**Review by dannylwe — APPROVED** [review]
LGTM

**Comment by adesege** [comment]
Did you also configure review apps and enabled pipeline development on Heroku? Pls share a screenshot of the pipeline page on Heroku

**Review by dorothyas — APPROVED** [review]

**Review by ezrogha — APPROVED** [review]
LGTM

**Comment by adesege** [comment]
From the screenshot, it says you should connect to Github to enable review apps. Can you confirm if you have enabled this feature?

**Comment by adesege** [comment]
Please rebase with develop

**Review by adesege — CHANGES_REQUESTED** [review]
You have not created a pipeline to create a deployment on the master branch. Please check the board for the acceptance criteria.

**Comment by felixkiryowa** [comment]
> You have not created a pipeline to create a deployment on the master branch. Please check the board for the acceptance criteria.

I worked  on this feedback

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: node server.js
```
Comment (adesege) [inline Procfile]: Change this to `web: npm start` instead

**Review by adesege — CHANGES_REQUESTED** [review]

File: Procfile (hunk 1 above)
Comment (felixkiryowa) [reply Procfile]: I have worked on the feedback

**Comment by adesege** [comment]
You have merge conflicts. @felixkiryowa

**Review by adesege — APPROVED** [review]

**Pull request merged**